src/Symfony/Bundle/FrameworkBundle/Resources/config/session.php

```diff
@@ -65,12 +68,12 @@
             ])
 
         ->set('session.flash_bag', FlashBag::class)
```

Comment:
I suppose we should not allow injecting a FlashBag anymore, right? This service is currently aliased to `Symfony\Component\HttpFoundation\Session\Flash\FlashBagInterface`.

Reply:
Yes, the service `session.flash_bag` triggers a deprecation (defined 2 lines below).
The alias `FlashBagInterface::class` does not trigger a deprecation it-self, but because it is an alias of `session.flash_bag` user will get the deprecation `The "session.flash_bag" service is deprecated, use "$session->getFlashBag()" instead.`.

Adding a `deprecate` notice on the alias leads to 2 deprecation (one for the alias, one for the service aliased).

Is it what your worried about with your comment?